tests/fixed/doc-comment-spacing.php

```diff
@@ -56,10 +56,14 @@ public function c(iterable $foo): void
      * @PHPCR\Uuid
      * @PHPCR\Field
      *
-     * @param int[] $foo
-     * @param int[] $bar
+     * @param         int[] $foo
+     * @param         int[] $bar
+     * @psalm-param   array<int, string> $bar
+     * @phpstan-param array<int, string> $bar
```

Comment:
Since there are only 2 parameters here, it's impossible to tell if the `@psalm-` and `@phpstan-` annotations are last because `$bar` is the last parameter, or because they are `@psalm-` and `@phpstan-` annotations. Also this snippet shows that types are aligned, but variable names are not, and does not show happens to descriptions.

Reply:
Entries are sorted in a group by description of the tool. I don't think we have to test this but I could add it. Variables aren't aligned in the group at least not beyond same annotation name. I don't know what to do about it. The phpcs + extension used here is not good documented and might have conflicting settings.
Maybe someone with more insights of the tool wanna have a look at it if this is a requirement.

Reply:
Can you annotate `$foo` instead of `$bar`? That will take care of my first remark. Also, `int[]` is not compatible with `array<int, string>`, it would be less confusing to have `array<int, int>` or array<string, int>` here.

Regarding the second remark, I don't think it should be a requirement to have the variables and descriptions aligned, and I thought it shouldn't come as a surprise to reviewers that would approve this, but maybe what you are saying is that it would be wrong to "enforce" not aligning with a test.

Also, it looks pretty hardcoded, so there is not much hope to get it fixed quickly:

https://github.com/squizlabs/PHP_CodeSniffer/blob/2bd19c2e6edb918ab14d87f1c4b37d1f112d8f00/src/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php#L296

Let's drop that second remark.

Reply:
@greg0ire I did the change in the test. Did you expect a correction of the param order? This did not happen and seems not to be part of the tool (or it had to be configured.

Reply:
@simonberger I was unsure and wanted to see what happened, as well as other reviewers. IMO it's already fine like this.